docs/user_authentication/third_party_token_management.md

```diff
@@ -0,0 +1,97 @@
+
+```sequence {theme: 'simple'}
+title Third-party Token Management
+
+participant Pbench-Server
+participant Browser
+participant Identity-Broker
+participant Identity-Provider
+
+autonumber 1
+activate Browser #red
+
+rbox over Browser: Dashboard
+Browser->Pbench-Server: GET Pbench client ID
+
+Pbench-Server->Browser: 200 Response 
+
+note right of Pbench-Server:{Identity_Broker_auth_URI: <auth_URI>\nclient_id: <pbench_client_id>\nclient_secret: <pbench_client_secret> # optional\n}
+
+note over Browser:User clicks login
+abox over Browser: Dashboard instructs the browser to \nload identity broker authentication page \nurl that was supplied by the Pbench-server.
+
+deactivate Browser
+
+Browser->Identity-Broker:GET identity broker auth URI\n(Authentication Request)
+
+note right of Browser:GET request:\n<identity_broker_auth_URI>\n?client_id=<pbench_client_id>\n&response_type=code\n&redirect_uri=<dashboard_URI>\n&scope=openid
```

Comment:
You should down-case `Identity_Broker_auth_URI` at line 18 to match what you've done at line 27 (`identity_broker_auth_URI`) and elsewhere.